RELEASING.md

```diff
@@ -24,95 +24,211 @@ tools must be installed.
-`docs.janusgraph.org` repo which you will update later.
+Use the [`docs/build-and-copy-docs.sh`](docs/build-and-copy-docs.sh) script to build a set of docs for this release and copy them to the cloned `docs.janusgraph.org` repo which you will update later.
+**NOTE:** the `docs/build-and-copy-docs.sh` script by default expects the janusgraph and docs.janusgraph.org repositories to be located at `~/github/janusgraph/janusgraph` and `~/github/janusgraph/docs.janusgraph.org` respectively.
+It is important that the the `latest` symbolic link at the root of docs.janusgraph.org is pointing to the correct version as well.
 
-You may also need to update the following files in the main repo for any new
-or updated dependencies:
+You may also need to update the following files in the main repo for any new or updated dependencies:
 
 * `NOTICE.txt`
 
-### Preparing the Local Repository
-
-Commit or stash any uncommitted changes.
-
-Recommended but not required:
-
-* Delete untracked files and directories in the clone by running
-  <code>git clean -fd</code>
-
-### Deploying Artifacts & Archives
-
-```bash
-# This script does several things.
-#
-# * Prompts for a ${janusgraph.compatible.versions} update
-# * Locally commits the release using the release plugin
-# * Deploys Maven artifacts to Sonatype OSS (staging, not released yet)
-# * Uploads zipfiles to S3
-# * Locally commits gh-pages updates (index.html and javadocs)
-# * Uploads AsciiDoc-generated documentation to S3
-#
-# Although it uploads to Sonatype OSS Staging and S3, it does
-# not push to github nor does it release the Sonatype repo.
-# This step is still essentially reversible: just destroy the
-# local commit history, drop the Sonatype OSS Staging repo, and
-# delete the files uploaded to S3.
-janusgraph-dist/src/release/release.sh
+### Testing
+By default Travis CI only runs tests against a single version of Cassandra.
+In order to run compatibility testing on all version of Cassandra listed as supported you can add [full build] to you commit message.
+
+### Versions
+
+This command will create a commit that updates the version from `x.y.z-SNAPSHOT` to `x.y.z`.
+The current `pom.xml` files will be backed up with the name `pom.xml.releaseBackup` and left in staging.
+
+```Shell
+mvn clean release:prepare
+```
+
+### Pull Request
+
+Open up pull requests for the documentation and version updates.
+After the updates are approved and merged, continue on with the release process.
+
+
+### GPG Signing Key
+
+The JanusGraph artifacts are signed with a GPG signature.
+If you don't already have a GPG signing key included with the `KEYS` file, you will need to create one and update the `KEYS` file.
+The `KEYS` file is posted on the releases page.
+
+Build Release Artifacts
+-----------------------
+
+* Pull down the latest, merged code from GitHub.
+* Stash any uncommitted changes.
+* Delete untracked files and directories.
+* Deploy it.
```

Comment:
While reading this, I'm not sure how this step works. How do I deploy it? The detailed steps below seem to end with release artifacts available locally in `~/jg-staging/`.

Reply:
That's exactly what it does. I can add in more detail about uploading artifacts to the release draft.